ModelicaReference/package.mo

```diff
@@ -1845,8 +1847,157 @@ The external C-functions may be defined in the following way:
   class 'function' "function"
     extends ModelicaReference.Icons.Information;
 
+ 
+
+class 'function partial application' "function partial application"
+    extends ModelicaReference.Icons.Information;
+
     annotation (Documentation(info="<html>
 <p>
+A function partial application is a function call with certain
+formal parameters bound to expressions. A function partial application
+returns a partially evaluated function that is also a function,
+with the remaining not bound formal parameters still present in the
```

Comment:
"not bound" -> "unbound" perhaps?

Reply:
Maybe at some time, but it corresponds to the 3.4 specification that we describe, so I don't think we should change it here (especially since it seems like a minor stylistic issue).